release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1080,6 +1080,78 @@ With this release, the CCO no longer checks for the nonexistent permission.
+
+* Previously, when installing a cluster on {vmw-full} with static IP addresses, the cluster could create control plane machines without static IP addresses due to a conflict with other Technology Preview features. With this release, the Control Plane Machine Set Operator correctly manages the static IP assignment for control plane machines. (link:https://issues.redhat.com/browse/OCPBUGS-29114[*OCPBUGS-29114*])
+
+* Previously, when installing a cluster on {gcp-short} with user-provided DNS, the installation program still attempted to validate DNS within the {gcp-short} DNS network. With this release, the installation program does not perform this validation for user-provided DNS. (link:https://issues.redhat.com/browse/OCPBUGS-29068[*OCPBUGS-29068*])
+
+* Previously, when deleting a private cluster on {ibm-cloud-name} that used the same domain name as a non-private {ibm-cloud-name} cluster, some resources were not deleted. With this release, all private cluster resources are deleted when the cluster is destroyed. (link:https://issues.redhat.com/browse/OCPBUGS-28870[*OCPBUGS-28870*])
+
+* Previously, when installing a cluster using a proxy with a character string using "%" in the configuration string, the cluster installation fails. With this release, the installation program correctly validates proxy configuration strings containing "%". (link:https://issues.redhat.com/browse/OCPBUGS-27965[*OCPBUGS-27965*])
+
+* Previously, the installation program still allowed the use of the `OpenShiftSDN` network plugin even though it was removed. With this release, the installation program correctly prevents installing a cluster with this network plugin. (link:https://issues.redhat.com/browse/OCPBUGS-27813[*OCPBUGS-27813*])
+
+* Previously, when installing a cluster on AWS Wavelengths or Local Zones into a region that supports either Wavelengths or Local Zones, but not both, the installation failed. With this release, installations into regions that support either Wavelengths or Local Zones can succeed. (link:https://issues.redhat.com/browse/OCPBUGS-27737[*OCPBUGS-27737*])
+
+* Previously, when a cluster installation was attempted using the same cluster name and base domain as an existing cluster, and the installation failed due to DNS record set conflicts, destroying the second cluster would also destroy the DNS record sets in the original cluster. With this release, the stored metadata contains the private zone name rather than the cluster domain, so only the correct DNS records are deleted upon cluster destruction. (link:https://issues.redhat.com/browse/OCPBUGS-27156[*OCPBUGS-27156*])
```

Comment:
Same comment as the previous one for `destroy`.
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#d

Reply:
:heavy_check_mark: